source/connection-monitoring-and-pooling/tests/pool-clear-close-in-use.json

```diff
@@ -0,0 +1,73 @@
+{
+  "version": 1,
+  "style": "unit",
+  "description": "Pool clear MUST support lazily closing in use connections",
+  "poolOptions": {
+    "backgroundThreadIntervalMS": 100
+  },
+  "operations": [
+    {
+      "name": "ready"
+    },
+    {
+      "name": "checkOut"
+    },
+    {
+      "name": "checkOut"
+    },
+    {
+      "name": "clear",
+      "closeInUseConnections": true
+    },
+    {
+      "name": "waitForEvent",
```

Comment:
by this waiting, I just wanted to highlight that `ConnectionClosed` and `ConnectionPoolCleared` happen approximately in the same time, but it's not important from test idea POV, thoughts?

Reply:
Yeah it's not strictly necessary, but it also doesn't hurt to include it.